package.json

```diff
@@ -7,17 +7,20 @@
     "url": "https://github.com/openfisca/openfisca-doc.git"
   },
   "scripts": {
+    "prebuild": "npm run clean",
     "build": "gitbook build",
-    "clean": "rm -rf _book",
+    "clean": "del _book",
+    "deploy": "npm run build && npm run deploy:rsync && git fetch",
```

Comment:
I run `git fetch` at the end because I want to have on my local git repository the actual status of the `gh-pages` branch. Sometimes I look at the `gh-pages` branch to know what's deployed, I see something weird, and D'OH! I forget to do `git fetch` so I see an image of the past :-/